### DISCUSSION TITLE
User Authentication (RBAC)

### DISCUSSION THREAD

**Discussion by zirota** [milvus-io/milvus#26455] [Q&A and General discussion]
Hello, understand that Milvus supports user authentication with RBAC.

Is there a way for me to link it up to LDAP or OpenID connect?

**Comment by xiaofan-luan** [comment]
> 

Hi @zirota 
We don't have plan to integrate LDAP as a opensource project. This is on our roadmap of our Self hosted offereings.

If you are interested to do it yourself you can try to integrate a hook on milvus and work together with your LDAP system

**Comment by MR-GOYAL** [comment]
Hi @zirota,

Have you proceeded on LDAP integration?

**Comment by CyanMystery** [comment]
> Hello, understand that Milvus supports user authentication with RBAC.
> 
> Is there a way for me to link it up to LDAP or OpenID connect?
Hello, may I ask which solution you chose in the end?
